apps/bmd/src/AppRoot.tsx

```diff
@@ -553,10 +556,13 @@ const AppRoot: React.FC<Props> = ({
 
   const resetBallot = useCallback(
     (showPostVotingInstructions?: PostVotingInstructions) => {
-      storage.remove(activationStorageKey)
-      storage.remove(votesStorageKey)
-      dispatchAppState({ type: 'resetBallot', showPostVotingInstructions })
-      history.push('/')
+      const resetStorageAndReset = async () => {
+        await storage.remove(activationStorageKey)
+        await storage.remove(votesStorageKey)
+        dispatchAppState({ type: 'resetBallot', showPostVotingInstructions })
+        history.push('/')
+      }
+      resetStorageAndReset()
```

Comment:
This drops the resulting `Promise`, so we're not handling a potential error here. Why not make `resetBallot` async?